.github/workflows/cw-check.yaml

```diff
@@ -32,8 +32,8 @@ jobs:
       - uses: DeterminateSystems/nix-installer-action@main
       - uses: DeterminateSystems/magic-nix-cache-action@main
         with:
-          is_ci : true
```

Comment:
btw, I don't know why we have this option - as it is not a valid option. also it is [not present at their README](https://github.com/DeterminateSystems/magic-nix-cache-action?tab=readme-ov-file#action-options).

Github actions use [`CI` env var to detect CI runtime](https://docs.github.com/fr/actions/learn-github-actions/variables#default-environment-variables).